### PR TITLE
DM-27414: Rename Hexapod to MTHexapod and Rotator to MTRotator

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: tests/test_config_files.py [hunk 1]
```diff
@@ -36,9 +36,9 @@ def test_MTDome(self):
             config_package_root=self.config_package_root,
         )
 
-    def test_Hexapod(self):
+    def test_MTHexapod(self):
         self.check_standard_config_files(
-            sal_name="Hexapod",
+            sal_name="MTHexapod",
             module_name="lsst.ts.hexapod",
```
Comment (wvreeven) [inline tests/test_config_files.py:42]: Shouldn't the module name also be updated to lsst.ts.mthexapod?

File: tests/test_config_files.py [hunk 2]
```diff
@@ -66,9 +66,9 @@ def test_MTMount(self):
             config_package_root=self.config_package_root,
         )
 
-    def test_Rotator(self):
+    def test_MTRotator(self):
         self.check_standard_config_files(
-            sal_name="Rotator",
+            sal_name="MTRotator",
             module_name="lsst.ts.rotator",
```
Comment (wvreeven) [inline tests/test_config_files.py:72]: Shouldn't the module name also be updated to lsst.ts.mtrotator?

**Review by wvreeven — APPROVED** [review]
Please review my two questions regarding the module names and fix if you agree. Also next time you may want to consider using "git mv" instead of a file system mv. In this case it is not important since there probably wasn't much git history for these files. Apart from that all looks good to me.

File: tests/test_config_files.py (hunk 2 above)
Comment (r-owen) [reply tests/test_config_files.py]: Yes. Good catch. (I updated this package before the rename and forgot to go back).

I fixed that and checked that the tests run.

**Pull request merged**